src/core/core/managers/pre_seed_data.py

```diff
@@ -201,14 +201,14 @@
         "name": "EMAIL Publisher",
         "description": "Publisher for publishing by email",
         "parameters": [
-            "SMTP_SERVER",
-            "SMTP_SERVER_PORT",
-            "EMAIL_USERNAME",
-            "EMAIL_PASSWORD",
-            "EMAIL_RECIPIENT",
-            "EMAIL_SUBJECT",
-            "EMAIL_MESSAGE",
-            "EMAIL_ENCRYPTION",
+            {"parameter": "SMTP_SERVER_ADDRESS", "rules": "required"},
+            {"parameter": "SMTP_SERVER_PORT"},
+            {"parameter": "SERVER_TLS", "type": "switch"},
+            {"parameter": "EMAIL_USERNAME"},
+            {"parameter": "EMAIL_PASSWORD"},
+            {"parameter": "EMAIL_SENDER", "rules": "email"},
+            {"parameter": "EMAIL_RECIPIENT", "rules": "email"},
```

Comment:
I think sender & recipient should be required fields.

Reply:
ideally it would be both